specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/managedClusters.json

```diff
@@ -3001,6 +3002,10 @@
               {
                 "value": "kubenet",
                 "description": "Use the Kubenet network plugin. See [Kubenet (basic) networking](https://docs.microsoft.com/azure/aks/concepts-network#kubenet-basic-networking) for more information."
+              },
+              {
+                "value": "none",
+                "description": "Use no CNI plugin pre-installed. See [BYO CNI](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni) for more information."
```

Comment:
FYI this is slightly different description from what we have in preview CLI, because before there were no public docs. Will update preview CLI versions to match in a subsequent PR.